graphene_plugin.py

```diff
@@ -333,7 +362,7 @@ def resolver_and_attribute_type_check(ctx: ClassDefContext) -> None:
                         continue
 
                     matching_resolver_argument = matching_resolver_arguments[0]
-                    if not are_types_equal(matching_resolver_argument.type, arg.type):
+                    if not is_equivalent(matching_resolver_argument.type, arg.type):
```

Comment:
Can you update the README to reflect this change? Particularly this part that claims these types need to match perfectly: 

>Erroring when the types defined in field attribute Arguments don't match up with type annotations in resolvers perfectly.

Reply:
I think the README here is still correct. `is_equivalent` is basically the same as the `==` I was using before except it includes the `Any`-checking we were manually doing before.

Reply:
But I did update the README to remove the bit about not handling return types! 😄